packages/prisma/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "clean": "rm -rf .turbo && rm -rf node_modules",
+    "build": "yarn prisma migrate deploy || true",
```

Comment:
This will attempt to migrate before build if there's a direct URL set. If not it will fail and move on with the build.